.coveragerc

```diff
@@ -0,0 +1,4 @@
+[run]
+  source = serenata_toolbox
+  omit =
```

Comment:
After removing the `omit` section, the results remained the same:

<img width="1110" alt="screen shot 2017-05-24 at 3 13 54 pm" src="https://cloud.githubusercontent.com/assets/4705127/26418842/12b55624-4094-11e7-90a9-88f01f655ca9.png">

I guess it is not necessary to keep such section by now. 😃